runtime/compiler/control/CompilationThread.cpp

```diff
@@ -11658,6 +11658,10 @@ TR::CompilationInfoPerThreadBase::processExceptionCommonTasks(
                                  _methodBeingCompiled->getMethodDetails().getMethod(),
                                  translationTime,
                                  compilationErrorNames[_methodBeingCompiled->_compErrCode]);
+            if (_methodBeingCompiled->_compErrCode == compilationHeapLimitExceeded)
+               {
+               TR_VerboseLog::write("\n OptPass:%d, VmState:%s", compiler->getOptIndex(), vmThread->omrVMThread->vmState);
```

Comment:
vmState is of type uintptr_t and here it is printed as a string.

Reply:
Please do not start a new line. Each compilation failure should be appear on a single line that starts with !
I suggest to replace `:` with `=` to be similar to other tokens on the line.